### ISSUE TITLE
Use RFC5322 Timestamp Format

### ISSUE THREAD

**Issue by corygalyna** [ZORALab/Hestia#351]
# Description
```
Please provide a short description of what you have encountered below.
```
Appearently, Debian 12 uses a different time stamp format for its default presentation. We best use `RFC 5322 format` like `Mon, 14 Aug 2006 02:34:56 -0600`.

Command is: `date --rfc-email`.

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```
Consistent date format usage.

# Current Behavior
```
Please specify the current behavior of software below.
```
very custom format.

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by corygalyna** [comment]
@hollowaykeanho , I know we're about to release but can we include this feature as well? It's non-backward compatible. Debian 12 (bookworm) produced a different date format causing inconsistencies.

**Comment by hollowaykeanho** [comment]
I didn't notice that... yeah but it will have slight delay. It should be included and stick to a consistent format.

**Comment by corygalyna** [comment]
TQ.

**Comment by hollowaykeanho** [comment]
I will do this after every pages are finalized and published first. It's a breaking change.

**Comment by hollowaykeanho** [comment]
Implemented in 6100b5311640e31bf6c7671764c6990704379d06

**Comment by hollowaykeanho** [comment]
Released in version `v.1.2.0`.
